### ISSUE TITLE
Mamba fails in some environments

### ISSUE THREAD

**Issue by djhoese** [astropy/ci-helpers#460]
Still tracking this down but I enabled mamba on a vispy PR I'm working on and seeing:

```
Executing transaction: ...working... done

./ci-helpers/travis/setup_dependencies_common.sh: line 55: mamba: command not found

./ci-helpers/travis/setup_dependencies_common.sh: line 428: mamba: command not found

./ci-helpers/travis/setup_dependencies_common.sh: line 55: mamba: command not found

./ci-helpers/travis/setup_dependencies_common.sh: line 55: mamba: command not found
```

Here's the full job: https://travis-ci.com/github/vispy/vispy/jobs/386465377

I can tell from the log that mamba is definitely installed, but something is getting lost. Perhaps a weird mix of conda failing and ci-helpers trying to do pip as a fallback.

**Comment by astrofrog** [comment]
I just ran into this for Windows builds.

**Comment by djhoese** [comment]
So I think I made some progress on this. I noticed that you (@astrofrog) pinned the conda version to 4.8.4 but 4.8.5 was just recently released. When mamba is installed it is using the newest conda which is 4.8.5, then a little later conda is re-installed to the pinned version (4.8.4 by default for mamba). This conda installation step seems to be what is uninstall mamba. However, I'm also noticing that it tries to install the non-mamba conda version 4.7.11 which shouldn't happen. Here's a full log:

https://travis-ci.com/github/pytroll/satpy/jobs/388314729

When I set my environment to `CONDA_VERSION=4.8.5` it all worked. So either the pin has to be relaxed (greater than or equal to 4.8?) or some other fix is needed. Maybe a different order of events for installing mamba and conda?
